### ISSUE TITLE
Add Windows 2025 support to Helix 

### ISSUE THREAD

**Issue by ilyas1974** [dotnet/dnceng#4170]
With the release of Windows 2025 approaching, we need to make sure we have Helix queues to support our customers workloads. 
Please create the following queues: 
windows.amd64.server2025.open
windows.amd64.server2025
~~windows.arm64.server2025.open~~
~~windows.arm64.server2025~~

### Release Note Category

- [x] Feature changes/additions
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements

### Release Note Description

Add `windows.server2025.amd64*` queues in Helix. New queues are very similar to existing `windows.amd64.server2022*` ones.

**Comment by ilyas1974** [comment]
A place to start with Windows 2025 - https://github.com/dotnet/dnceng/issues/3343

**Comment by missymessa** [comment]
Also need to have this work done before creating the new queues: https://github.com/dotnet/dnceng/issues/3781

**Comment by dougbu** [comment]
working on x64 queues only at the moment. no Marketplace ARM64 images available for Server 2025

queues could be named `Windows.Amd64.Server2025[.Open]` (as requested above) or `@Windows.Server2025.Amd64[.Open]`. existing   `Windows.Amd64.Server2022[.Open]` names argue for first choice. glaring counter-examples include `windows.11.amd64.clent[.open]`, `windows.server.amd64.vs2017*`, `windows.server.amd64.vs2019*`, and of course all of the Linux and macOS queue names that start with the OS and keep the OS bits together. what's your preference @ilyas1974❓

either way, new queues will start w/ the same artefacts as the `Windows.Amd64.Server2022[.Open]` ones

will of course add `...rt` and `...svc` queues too, avoiding future emergency reactions to customer requests

**Comment by ilyas1974** [comment]
Let's go with windows.server2025.amd64.* for the naming convention.

**Comment by PriyaPurkayastha** [comment]
Happy New Year folks! Checking back on status of these new queues - any ETA on when these will become available? 
cc @Junjun-zhao

**Comment by dougbu** [comment]
The issue on our side, Add Windows 2025 support to Helix * Issue #4170 * dotnet/dnceng<https://github.com/dotnet/dnceng/issues/4170>, is only waiting for our next rollout. That should occur on the 8th (next Wednesday).

Note we don't have Server 2025 images in our regular build pools, e.g., NetCore1ESPool-Internal, yet. That's tracked in Add Windows Server 2025 Build Agent Pools * Issue #3781 * dotnet/dnceng<https://github.com/dotnet/dnceng/issues/3781>.

thanx,
    doug

**Comment by dougbu** [comment]
ugh, replying to a GitHub comment in email really messes things up 😦  editing&hellip;

**Comment by Junjun-zhao** [comment]
Hi @dougbu @ilyas1974, Happy New Year!~

This issue seems to be related to the Server2025 queue. Will the new Windows 11 queues for our team be part of the next rollout?
- windows.11.amd64.coreappcompat
- windows.11.amd64.coreappcompat.pr

**Comment by dougbu** [comment]
no, we have not begun work on #4714. I just marked that issue as needing triage

**Comment by Junjun-zhao** [comment]
Got it. Thanks @dougbu.  Is there any ETA for https://github.com/dotnet/dnceng/issues/4714?

**Comment by dougbu** [comment]
done
